source/features/remove-projects-tab.tsx

```diff
@@ -31,38 +34,18 @@ const addNewProjectLink = onetime(() => {
 			New project
 		</a>
 	);
-});
+}
 
-async function init(): Promise<false | void> {
-	await elementReady(`
-		.orghead + *,
-		.repohead + *,
-		.user-profile-nav + *
-	`); // Wait for the tab bar to be loaded
-
-	const projectsTab = select([
-		'[data-hotkey="g b"]', // In organizations and repos
-		'.user-profile-nav [href$="?tab=projects"]' // In user profiles
-	]);
+async function removeProjectsTab(): Promise<void> {
+	const projectsTab = await getProjectsTab();
 
 	if (!projectsTab) {
 		// Projects aren't enabled here
 		return;
 	}
 
-	addNewProjectLink();
-
-	// If there's a settings tab, the current user can disable the projects,
-	// so the tab should not be hidden
-	if (select.exists([
-		'.js-repo-nav [data-selected-links^="repo_settings"]', // In repos
-		'.pagehead-tabs-item[href$="/settings/profile"]' // In organizations
-	])) {
-		return;
-	}
-
-	// Only remove the tab if it's not the current page and if it has 0 projects
-	if (!projectsTab.matches('.selected') && select('.Counter', projectsTab)!.textContent!.trim() === '0') {
+	// Only remove the tab if it's not the current page and if it has 0 projects or on `isOrganizationProfile` it will be blank
+	if (!projectsTab.matches('.selected') && ['0', ''].includes(select('.Counter', projectsTab)!.textContent!.trim().charAt(0))) {
```

Comment:
the `charAt(0)` make sure we dont include 10 or 100 etc

Reply:
Why this change? Doesn’t seem necessary and it’s not a good solution

Reply:
On an org page if there are any projects it will display a number in the counter. If not the counter textcontent will be blank. 
On a repo page the counter will be 0 if there are no projects.

Reply:
> If not the counter textcontent will be blank.

Not true. Does this work at all on orgs? There's no `.Counter` element there, this code should error.

https://github.com/babel

<img width="255" alt="" src="https://user-images.githubusercontent.com/1402241/82239979-ec029800-9939-11ea-8614-0b87aa23eb30.png">

https://github.com/microsoft

<img width="383" alt="" src="https://user-images.githubusercontent.com/1402241/82240036-05a3df80-993a-11ea-963c-54995ed54835.png">

Maybe:

```js
const counter = select('.Counter', projectsTab);
if (!projectsTab.matches('.selected') && (!counter || counter.textContent!.trim() === '0')) {
  return false;
}
```